src/README.rst

```diff
@@ -27,7 +27,9 @@ Unreleased
 - Add tests to verify correctness of help text (#71)
 - Add scaling policy parameter to the command for service create and the command for service update (#76)
 - Add new group called container with commands: invoke-api (invoke raw container REST API) and logs (get container logs) (#82)
-- Add support for passing json values to arguments as .txt files (#84)
+- Add support for passing json values to arguments as .txt files. Input starting with '@' are considered paths (#84)
+- Update to 6.2 Service Fabric runtime (#97)
```

Comment:
Can you change the name if that's the goal of this PR, `Reformat function calls` seems misleading :P